addons/addon-environment-sc-ui/packages/environment-type-mgmt-ui/src/parts/environment-types/EnvTypeEditor.js

```diff
@@ -165,6 +165,7 @@ class EnvTypeEditor extends React.Component {
       case 'configurations':
         return (
           <ConfigStep
+            data-testid="configstep"
```

Comment:
Can we change these `ids` as well?

Reply:
yup to all of these--sorry I didn't realize there were more!